src/cryptonote_protocol/CMakeLists.txt

```diff
@@ -33,11 +33,11 @@ add_library(cryptonote_protocol
   core_protocol_instantiation.cpp
   p2p_net_node_instantiation.cpp
 )
-
 target_link_libraries(cryptonote_protocol
   PUBLIC
     p2p
   PRIVATE
+    common
```

Comment:
This should not be necessary: anything that links to p2p already gets a (public) link to `cryptonote_core` which itself has a public link to `common`.

Reply:
I think that:
```
target_include_directories(target PUBLIC $<TARGET_PROPERTY:lokimq,INCLUDE_DIRECTORIES>)
```
may work.  We should probably move that one up into the public include directories of the common target.

Reply:
Hmm, `common` should *already* have that include directory (and pass it on to anything that links against common)

Reply:
Ah, mnemonics doesn't link against common.  That's probably the easiest fix.